.github/CODEOWNERS

```diff
@@ -17,7 +17,13 @@
 # Sharmila Ramamoorthy (sharmilarama)
 # Evgeny Uglov (EvgenyUglov)
 # Bowen Jiang (bjiang27)
+# Tao He (taohe1012)
+# Peter Cao (P-Cao)
+# Yiming Bao (baoy1)
+# Yian Zong  (YianZong)
+# Forrest Xia (forrestxia)
 
 
 # for all files:
```

Comment:
Please remove Logan Jones (lj-software) and Michael Edegware @medegw01

Reply:
@hoppea2 updated, please check and give your approval. thanks!